tooling/template-base/components/Navigation/CurrentUserNavBar.tsx

```diff
@@ -54,17 +54,17 @@ export default function CurrentUserNavBar() {
                 <Avatar size="small" />
                 <div className="mx-3 my-auto">
                   <button className="hover:text-blue-700">
-                    <a href="/learn/profile">My Profile</a>
+                    <a href="/learn/account?tab=dashboard.account_profile">My Profile</a>
                   </button>
                 </div>
               </div>
               <hr></hr>
-              <a href="/learn/profile">
+              <a href="/learn/account?tab=dashboard.account_profile">
```

Comment:
Should this one be linked to `/learn/account?tab=dashboard.account` as seen in `CurrentUserSmallScreenNavBar`?